crucible-llvm/src/Lang/Crucible/LLVM/Translation/BlockInfo.hs

```diff
@@ -212,6 +224,10 @@ instrUse from i bim = Set.unions $ case i of
   L.Bit _op x y -> [useTypedVal x, useVal y ]
   L.Conv _op x _tp -> [useTypedVal x]
   L.Call _tailCall _tp f args -> useVal f : map useTypedVal args
+  L.CallBr _tp f args norm ls ->
+    [useVal f, useLabel from norm bim] ++
+      map (\l -> useLabel from l bim) ls ++
+      map useTypedVal args
```

Comment:
The case for `Invoke` has this comment:

```
-- NB, this is only correct for "invoke" instructions that don't assign the return value
```

I think we'd want a similar comment for `CallBr` as well, given that the indirect labels don't make use of the returned value.

Reply:
Gotcha, to be frank I didn't quite understand what this comment left, but I'll apply it here as well.